app/templates/components/orders/event-info.hbs

```diff
@@ -14,12 +14,12 @@
   <div class="ui padded segment">
     <strong>At {{data.event.locationName}}</strong>
     <br>
-    <strong>From :</strong> {{header-date data.event.startsAt}}
+    <strong>From:</strong> {{header-date data.event.startsAt}}
```

Comment:
Put all these strings inside translation helpers.

Reply:
@CosmicCoder96 Do you mean the files in `translations/`?

Reply:
@uds5501 , no not the files in translations.
Instead of writing text simply you've to write it like
{{t 'From: '}}
This is what is meant by translation helpers.

Reply:
@ritikamotwani Okay! Applied the changes, please review now